os/Makefile.unix

```diff
@@ -519,11 +519,25 @@ post:
 ifeq ($(CONFIG_APP_BINARY_SEPARATION),y)
 	$(Q) mkdir -p $(OUTBIN_DIR)/user
 ifneq ($(CONFIG_SUPPORT_COMMON_BINARY),y)
+ifeq ($(CONFIG_APP_INFO),y)
+	$(Q) $(call VERIFY_APP,"app")
```

Comment:
There are also differences between public repo and private repo for using CONFIG_APPX_XXX. For example, in `os/tools/set_bininfo.py`.
Could you integrate them also if you want to use common code for all repos?

Reply:
Done.